components/Card/index.js

```diff
@@ -0,0 +1,75 @@
+import { useAtom } from "jotai";
+import { useState } from "react";
+
+export default function Card({ initialItems, items }) {
+  const [searchInput, setSearchInput] = useState("");
+  const [selectedItems, setSelectedItems] = useAtom(initialItems);
+
+  const filteredItems = items.tags.filter((item) =>
+    item.name.toLowerCase().includes(searchInput.toLowerCase())
+  );
+
+  return (
+    <>
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          const formData = new FormData(event.target);
+          const data = Object.fromEntries(formData);
+          setSearchInput("");
+          const selectedItem = filteredItems.find(
+            (item) => item.name === data.search
+          );
+
+          if (
+            //makes reselection impossible
+            selectedItem &&
+            !selectedItems.find((item) => item.name === selectedItem.name)
+          )
+            setSelectedItems([...selectedItems, selectedItem]);
+        }}
+      >
+        <input
```

Comment:
For semantic and accessibility reasons, this `input` should have a `label`.

Reply:
ok i added a label now ..thanks